### PR TITLE
Bump Laravel versions

### PR DESCRIPTION
**Pull request by barryvdh**
Let's drop old Laravel versions and start testing Laravel 8

### CODE REVIEW

**Comment by mfn** [comment]
# YES

**Comment by barryvdh** [comment]
Wel technically 5.5 is still supported for two days :P https://laravel.com/docs/7.x/releases
But it's about time.

**Comment by barryvdh** [comment]
Hmm so dependencies are all good, but not sure why it's failing 1 test.

**Comment by mfn** [comment]
> Hmm so dependencies are all good, but not sure why it's failing 1 test.

The test `\Barryvdh\LaravelIdeHelper\Tests\ExampleTest::testEloquentBuilderOutput` uses reflection to get phpdoc of `\Illuminate\Database\Eloquent\Builder::with` which has changed between 7/8

I don't know this code part how relevant this is (remove test? make a condition to check between L7/L8? 🤷‍♀️)

**Comment by barryvdh** [comment]
Yeah nog sure, maybe update for Laravel 8 and skip the rest?

**Comment by barryvdh** [comment]
Alright, like this then?

File: tests/MethodTest.php [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 use Barryvdh\LaravelIdeHelper\Method;
 use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Foundation\Application;
 use PHPUnit\Framework\TestCase;
 
-class ExampleTest extends TestCase
+class MethodTest extends TestCase
```
Comment (mfn) [inline tests/MethodTest.php:10]: 😅

This codebase...

**Review by mfn — APPROVED** [review]
👍

**Pull request merged**